packages/css/src/breadcrumbs.css

```diff
@@ -5,7 +5,9 @@
   --dsc-breadcrumbs-color: var(--ds-color-text-subtle);
 
   & > :is(ol, ul) {
-    display: flex;
+    &:not([hidden]) {
+      display: flex;
+    }
```

Comment:
To keep in line with how we do nesting other places, I think this should be moved below the other properties